.github/workflows/deploy.yml

```
@@ -1,87 +1,78 @@
-    permissions:
-      contents: read
-      packages: write
-      
+    concurrency:
+      group: deploy-prod
+      cancel-in-progress: false
+
     steps:
       - name: Checkout code
-        uses: actions/checkout@v4
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)